www/docs/providers/azure-ad-b2c.md

```diff
@@ -12,7 +12,20 @@ https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-c
 https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-tenant
 
 ## Example
+- In https://portal.azure.com/ -> Azure Active Directory create a new App Registration.
+- Make sure to remember / copy
+  - Application (client) ID
+  - Directory (tenant) ID
+- When asked for a redirection URL, use http://localhost:3000/api/auth/callback/azure-ad-b2c
+- Create a new secret and remember / copy its value immediately, it will disappear.
 
+In .env create the follwing entries
```

Comment:
According to the Next.js [Environment Variables](https://nextjs.org/docs/basic-features/environment-variables) guidelines, local env variables should be added to `.env.local`, since `.env` is supposed to be added to version control. (It is especially important in case of `AZURE_CLIENT_SECRET`)

Reply:
Thanks for the hint, I filed another PR which covers that.